Homework/Homework/Program.cs

```diff
@@ -151,4 +153,14 @@ private static void WriteTextWithBorder(string text)
         Console.WriteLine($"| {text} |");
         Console.WriteLine($"+-{upAndDown}-+");
     }
+    public static int[] GetFirstEvenNumbers(int count)
+    {
+        int[] evenNumbers = new int[count];
+        for (int i = 0; i < count; i++)
```

Comment:
```suggestion
        for (int i = 0; i < evenNumbers.Length; i++)
```
И еще лучше всегда делать так